pandas/core/window/ewm.py

```diff
@@ -276,7 +257,7 @@ def mean(self, *args, **kwargs):
         Parameters
         ----------
         *args, **kwargs
-            Arguments and keyword arguments to be passed into func.
+            Keyword arguments to be passed into func.
```

Comment:
```suggestion
            Arguments and keyword arguments to be passed into func.
```
You can commit this from the UI.